content/en/security/identity_risks/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cloud Security Management Identity Risks
+kind: documentation
+further_reading:
+- link: "/security/cloud_security_management/"
+  tag: "Documentation"
+  text: "Learn more about Cloud Security Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+CSM Identity Risks is not available in this site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">CSM Identity Risks is in beta.</div>
+
+Cloud Security Management Identity Risks (CSM Identity Risks) provides in-depth visibility into your organization's AWS IAM risks. It enables you to detect and resolve identity risks on an ongoing basis to secure your cloud infrastructure from IAM-based attacks.
```

Comment:
Let's remove AWS from the above line ("... into your organization's IAM risks.") as we are clarifying the current limitation (AWS only) below?